detox/src/client/Client.js

```diff
@@ -336,7 +345,9 @@ class Client {
       this._asyncWebSocket.rejectAll(this._pendingAppCrash);
       this._pendingAppCrash = null;
     } else if (this._asyncWebSocket.hasPendingActions()) {
-      this._asyncWebSocket.rejectAll(new DetoxRuntimeError('The app has unexpectedly disconnected from Detox server.'));
+      this._asyncWebSocket.rejectAll(wasAppTerminatedByUs
+        ? new DetoxRuntimeError('The app has been unresponsive and Detox had to terminate it forcibly.')
```

Comment:
TODO: check if my English wording is good here 😅

Reply:
Disregard, I think it is too early to app termination to here. Let it be just stuck.